### PR TITLE
Update: Make SigmaDAO compatible with new updated trigger

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

File: src/indexer/index.ts [hunk 1]
```diff
@@ -40,11 +40,11 @@ export const getApplicationGlobalState = async (
 		const globalStateRes = await executeReq(lookupApplications(appId));
 		if (
 			globalStateRes?.allSigmaDaos?.nodes.length &&
-			JSON.parse(globalStateRes.allSigmaDaos.nodes[0].appParams).dt.gd
+			JSON.parse(globalStateRes.allSigmaDaos.nodes[0].appParams).gs
```
Comment (Megha-Dev-19) [inline src/indexer/index.ts:43]: what's `dt.gd` and `gs`.?

**Review by Megha-Dev-19 — COMMENTED** [review]
Why are we changing `decodeAppLocalState` to  `decodeAppParamsState`.?

File: src/indexer/index.ts (hunk 1 above)
Comment (rmeena840) [reply src/indexer/index.ts:43]: json structure changed. it's now gs

**Comment by rmeena840** [comment]
@Megha-Dev-19 Can you approve it?

File: src/indexer/index.ts (hunk 1 above)
Comment (robert-zaremba) [reply src/indexer/index.ts:43]: please add a comment explaining what is `gs`.

**Review by robert-zaremba — APPROVED** [review]
please add some comments to the code

File: src/indexer/index.ts (hunk 1 above)
Comment (rmeena840) [reply src/indexer/index.ts:43]: Done

**Pull request merged**